xla/service/elemental_ir_emitter_test.cc

```diff
@@ -101,6 +101,162 @@ ENTRY main {
   RunTest(hlo_text, {&lhs, &rhs});
 }
 
+inline std::string llvmValueToString(llvm::Value* v) {
```

Comment:
We have https://github.com/openxla/xla/blob/main/xla/service/llvm_ir/llvm_util.h#L61

Reply:
replaced `llvmValueToString` with `llvm_ir::DumpToString`